### ISSUE TITLE
Грешна препратка към repo-то

### ISSUE THREAD

**Issue by xswordsx** [obshtestvo/sledi-parlamenta#57]
Препратката в секция [Кой](http://www.obshtestvo.bg/project/openparliament.html#who), която трябва са води към това repo води към [obshtestvo/rating-gov-representatives](https://github.com/obshtestvo/rating-gov-representatives)

**Comment by antitoxic** [comment]
Да. Строшен линк. Трябва да се мине през сайта и да се разгледа. https://github.com/obshtestvo/obshtestvo.bg/issues/33
